wls-common/security/src/test/java/de/muenchen/oss/wahllokalsystem/wls/common/security/EncryptionBuilderTest.java

```diff
@@ -0,0 +1,84 @@
+package de.muenchen.oss.wahllokalsystem.wls.common.security;
+
+import static org.assertj.core.api.Assertions.fail;
+import de.muenchen.oss.wahllokalsystem.wls.common.security.testultils.LoggerExtension;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import javax.crypto.NoSuchPaddingException;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class EncryptionBuilderTest {
+
+    @RegisterExtension
+    public LoggerExtension loggerExtension = new LoggerExtension();
+
+    @Nested
+    class decrypt {
```

Comment:
Bitte auf die Namenskonvetion für Klassen achten. Ich würde die Klasse auch direkt nach der entsprechenden Methode benennen.

```suggestion
    class DecryptValue {
```